stdlib/test/bit/test_bit.mojo

```diff
@@ -12,11 +12,57 @@
 # ===----------------------------------------------------------------------=== #
 # RUN: %mojo %s
 
-from bit import rotate_bits_left, rotate_bits_right
+from bit import rotate_bits_left, rotate_bits_right, bit_ceil, bit_floor
 
 from testing import assert_equal
 
 
+def test_bit_ceil():
+    assert_equal(bit_ceil(-2), 1)
+    assert_equal(bit_ceil(1), 1)
+    assert_equal(bit_ceil(2), 2)
+    assert_equal(bit_ceil(4), 4)
+    assert_equal(bit_ceil(5), 8)
+
+
+def test_bit_ceil_simd():
+    alias simd_width = 4
+    alias type = DType.int8
+
+    alias var1 = SIMD[type, simd_width](-2, -1, 3, 4)
+    assert_equal(
+        bit_ceil[type, simd_width](var1), SIMD[type, simd_width](1, 1, 4, 4)
+    )
```

Comment:
here and below: please rely on parameter inference instead of passing explicitly
```suggestion
    assert_equal(
        bit_ceil(var1), SIMD[type, simd_width](1, 1, 4, 4)
    )
```